pandas/core/indexes/datetimelike.py

```diff
@@ -189,6 +190,30 @@ def format(
 
         return self._format_with_header(header, na_rep=na_rep, date_format=date_format)
 
+    def _format_array(
+        self,
+        formatter: Callable | None,
+        float_format: FloatFormatType = None,
+        na_rep: str = "NaN",
+        digits: int = None,
+        space: str | int = None,
+        justify: str = "right",
+        decimal: str = ".",
+        leading_space: bool | None = True,
+        quoting: int | None = None,
+    ):
+        return self.array._format_array(
```

Comment:
Is there an easier way to dispatch a method from the datetime-like index to the EA? Maybe @jbrockmendel has an idea?

Reply:
should be able to add it to the `inherit_names` decorator above

Reply:
Nevermind. We can just make sure to call `extract_array` earlier on.